src/main/scala/ai/privado/utility/Utilities.scala

```diff
@@ -424,117 +423,4 @@ object Utilities {
     })
     semanticList.toList
   }
-
-  def reportUnresolvedMethods(xtocpg: Try[Cpg], filename: String): Unit = {
```

Comment:
@gaurav-gogia did we remove this method?

Reply:
@pandurangpatil, this one is a duplicate. I forgot to remove it before by mistake. Correct one can be found here:
`src/main/scala/ai/privado/utility/UnresolvedReportUtility.scala`